src/db/tables/featureFlags.ts

```diff
@@ -73,6 +73,7 @@ export const featureFlagNames = [
   "EditScanProfileDetails",
   "SubPlat3",
   "Announcements",
+  "PrivacyProductsBundle",
```

Comment:
Created [MNTOR-4488](https://mozilla-hub.atlassian.net/browse/MNTOR-4488) for removing the feature flag after rollout.